CHANGELOG.md

```diff
@@ -1,3 +1,6 @@
+## v6.2.1
+* Fix SecurityTokenCacher to not cache tokens forever.
```

Comment:
was it really stored forever though? this `@cache` should only persist the time of one request?

good refactor getting rid of the mutex and unnecessary hash though. but I'm not sure the changelog entry is correct, or if it is, I don't understand where that was stored forever

Reply:
Please see the discussion on Slack, I'll send you the link there.

Reply:
got my answer by reading this thread https://mdsol.slack.com/archives/C2NUF7DNJ/p1631773294025100

thank you

Reply:
Thanks